requirements.txt

```diff
@@ -7,7 +7,6 @@ scipy
 typing-extensions
 altair
 xlrd
-keras>=2.3.1
```

Comment:
keras is nog wel nodig op de dgx, anders werkt from keras_applications.imagenet_utils import _obtain_input_shape niet

Reply:
Ik heb keras weer toegevoegd